robot/basestation/static/js/roslibjs/ros-helpers.js

```diff
@@ -641,24 +644,22 @@ function sendRequest (device, command, callback, timeout = REQUEST_TIMEOUT) {
 }
 
 function sendRoverCommand (cmd) {
+  logDebug('Sending "' + cmd + '" to Rover Teensy')
   let command = new ROSLIB.Message({ data: cmd })
-  console.log(command)
-  appendToConsole('Sending "' + cmd + '" to rover Teensy')
   rover_command_publisher.publish(command)
 }
 
 function sendPdsCommand (cmd) {
+  logDebug('Sending "' + cmd + '" to PDS Teensy')
   let command = new ROSLIB.Message({ data: cmd })
-  console.log(command)
-  appendToConsole('Sending "' + cmd + '" to PDS Teensy')
   pds_command_publisher.publish(command)
 }
 
 /*
 returnsthe IP portion of the currntly set ROS_MASTER_URI
 */
 function getRoverIP (callback) {
-  console.log('roverIP: ' + env.ROS_MASTER_IP)
-  console.log('hostIP: ' + env.HOST_IP)
+  logDebug('roverIP: ' + env.ROS_MASTER_IP)
+  logDebug('hostIP: ' + env.HOST_IP)
```

Comment:
What's your reasoning for setting these as debug but not the ones on top like GUI colour theme?

Reply:
The GUI color theme is information that an operator can understand. This is debug information that would be useful to only someone in the software team

Reply:
Perhaps we should try to indicate when exactly each log should be used.

How I see it 

Info : Information that is operator readable (Operator can be mech or elec person, and may have no knowledge or understanding of the codebase).
Debug : Information useful for the software team if something is wrong. We should avoid putting debug on everything.
Warn : Warns in a situation that shouldn't happen or a behavior that's not normal but it isn't necessarily causing an issue.
Error : This is just not normal
Fatal : Fuck

Reply:
generally i agree but since the ip address stuff only happens once i think it's fine to leave it as info, it's not a spammy kind of log message